porting/npl/freertos/include/nimble/nimble_npl_os.h

```diff
@@ -99,12 +99,24 @@ ble_npl_eventq_init(struct ble_npl_eventq *evq)
     evq->q = xQueueCreate(32, sizeof(struct ble_npl_eventq *));
 }
 
+static inline struct ble_npl_event *
+ble_npl_eventq_get_tmo(struct ble_npl_eventq *evq, ble_npl_time_t tmo)
+{
+    return npl_freertos_eventq_get_tmo(evq, tmo);
```

Comment:
this is no longer necessary - you can just define `ble_npl_eventq_get_tmo` in `.c` file without need to "jump" through `static inline` stub. we'll fix other calls later.

Reply:
We decided to not change this now and fix all calls later.